ani-cli

```diff
@@ -128,7 +125,7 @@ get_links() {
             ;;
         *) [ -n "$episode_link" ] && printf "%s\n" "$episode_link" ;;
     esac
-    printf "Fetching %s Links\n" "$provider_name" 1>&2
+    printf "\033[1;32m %s\033[0m Links Fetched\n" "$provider_name" 1>&2
```

Comment:
Good one. But now that you opened tha UI fixin box, I'm not stopping until the deleting of lines is consistent

Reply:
wouldn't just adding `tput clear` above this line be sufficient for that?